openssl-dynamic/src/main/java/io/netty/internal/tcnative/CertificateCallbackTask.java

```diff
@@ -35,15 +35,15 @@
 
     // See https://www.openssl.org/docs/man1.0.2/man3/SSL_set_cert_cb.html.
     @Override
-    protected int runTask(long ssl) {
+    protected void runTask(long ssl, TaskCallback taskCallback) {
```

Comment:
Not related with this PR, but do we have a runtime check for TCNative version, so our users get more friendly exception message in case of version mismatch?

Reply:
@trustin yes we have in netty itself.